R/data_frame.R

```diff
@@ -99,11 +94,7 @@ inspect_data_frame <- function(
   }
 
   if (is_na_value(df)) {
```

Comment:
makes little sense to check for NA specifically at this point. Just let it fail later in is.data.frame